app/serializers/organization_page_serializer.rb

```diff
@@ -0,0 +1,36 @@
+class OrganizationPageSerializer
+  include RestPack::Serializer
```

Comment:
can you use the restpack serializer wrapper here instead and setup preloading to avoid N+1 for linked resources.